homework/Repository.js

```diff
@@ -13,15 +13,36 @@ class Repository {
    * @param {HTMLElement} container The container element in which to render the repository.
    */
   render(container) {
-    // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, JSON.stringify(this.repository, null, 2));
+    const repositoryInformation = Util.createAndAppend('ul', container, {
+      id: 'repository-information',
+    });
+
+    const RepositoryName = Util.createAndAppend('li', repositoryInformation, { text: '' });
+    const description = Util.createAndAppend('li', repositoryInformation, { text: '' });
+    const forks = Util.createAndAppend('li', repositoryInformation, { text: '' });
+    const update = Util.createAndAppend('li', repositoryInformation, { text: '' });
+
+    RepositoryName.innerHTML = `Repository:<a target=_blank href='${this.repository.html_url}'> ${
+      this.repository.name
+    }</a>`;
+
+    description.innerHTML = `Description: ${this.repository.description}`;
+    forks.innerHTML = `Forks: ${this.repository.forks}`;
+    update.innerHTML = `Updated: ${this.repository.updated_at}`;
+
+    if (this.repository.description === null) {
+      description.style.display = 'none';
+    } else {
+      description.style.display = 'block';
+    }
   }
 
   /**
    * Returns an array of contributors as a promise
    */
-  fetchContributors() {
-    return Util.fetchJSON(this.repository.contributors_url);
+  async fetchContributors() {
+    const reso = await Util.fetchJSON(this.repository.contributors_url);
```

Comment:
Umm.. I don't think this is the best name for a variable here. 🙂

Reply:
Done !